packages/nextjs/app/builders/0x7429CbD5eD20736645723E972bE60B7F6BF5959c/page.tsx

```diff
@@ -0,0 +1,158 @@
+"use client";
+
+import { useCallback, useEffect, useState } from "react";
+import Image from "next/image";
+import Link from "next/link";
+import { Check, Code, Copy, Cpu, Github, Globe, Linkedin, Twitter } from "lucide-react";
```

Comment:
For Github, LinkedIn, Twitter it gives me deprecated warning. Maybe you want to consider to change to another library, like react icons.